server/web/src/vite-env.d.ts

```diff
@@ -1,296 +0,0 @@
-/// <reference types="vite/client" />
-
-type Settings = {
```

Comment:
I'm not sure what is the problem, VSCode complains about not finding this type but I can launch the project fine. 

Should we keep this type in this file?

Reply:
I don't know 🤷

Reply:
Readded the Settings object